src/logservice/logminer/ob_log_miner_record.cpp

```diff
@@ -864,18 +866,18 @@ int ObLogMinerRecord::build_where_conds_(ObStringBuffer &stmt,
 		binlogBuf *cols,
 		const unsigned int col_cnt,
 		ITableMeta *tbl_meta,
-    bool &has_lob_null,
+    bool &has_unreliable_null,
```

Comment:
please check all format

Reply:
Thank you for your suggestion. I have made the changes.